DataModel/ExperimentDesign/ExperimentDesign-definition.yml

```diff
@@ -7,6 +7,13 @@ ExperimentDesign:
       description: The organism under study, providing information about the species or biological system.
       format: NCBITaxon
       type: string
+    cellLine:
+      description: The cell lines from which the samples were derived for the dataset.
+      format: Cellosaurus
+      type: text
+    geneticModification:
+      description: The type of genetic modification done on the sample.
+      type: text
     totalNumberSamples:
```

Comment:
@ishitasati11 Can we add a limit for numerical attributes, as in, 
`lower limit` and `upper limit`. It would be nice to have it!